### PR TITLE
Hemera avs operator

### PR DESCRIPTION
**Pull request by zhufengthehemera**

### CODE REVIEW

File: indexer/modules/custom/avs_operator/operator.py [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+            kwargs["max_workers"],
+            job_name=self.__class__.__name__,
+        )
+        self._is_batch = kwargs["batch_size"] > 1
+        self._filters = kwargs.get("filters", [])
+        self.index_record_event = IndexRecordEvent()
+        # TODO: replace by dataclass.get_code_hash()
+        self.block_dataclass = hex_str_to_bytes("0x67a70c90")
+        self.tx_dataclass = hex_str_to_bytes("0x6048a7e2")
+        with open(self.user_defined_config["operator_ecdsa_key_file"], "r") as keystore_file:
```
Comment (shanshuo0918) [inline indexer/modules/custom/avs_operator/operator.py:47]: 这一部分是不是可以抽出来，专门写一个class或者function，就是key和签名相关的

File: indexer/modules/custom/avs_operator/operator.py [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+            verify_status=log.verified,
+            confirm_status=False,
+        )
+        hs.msg_hash = hs.message_hash()
+
+        if log.verified is False:
+            return hs
+
+        try:
+            task_resp: AlertTaskInfo = self.aggregator_client.create_alert_task(bytes_to_hex_str(hs.msg_hash))
```
Comment (shanshuo0918) [inline indexer/modules/custom/avs_operator/operator.py:211]: 这个如果多个 operator都create 这个task会怎样呀

Comment (zhufengthehemera) [reply indexer/modules/custom/avs_operator/operator.py:211]: aggregator 对同一个 task 会直接返回成功

Comment (zhufengthehemera) [reply indexer/modules/custom/avs_operator/operator.py:211]: `task := agg.GetTaskByAlertHash(req.AlertHash)
	if task == nil {
		agg.logger.Info("create new task", "alert", req.AlertHash)
		taskIndex := agg.newIndex()

		var err error
		task, err = agg.sendNewTask(req.AlertHash, taskIndex)

		if err != nil {
			agg.logger.Error("send new task failed", "err", err)
			return nil, err
		}
	} else {
		agg.logger.Info("the task had created", "task", task)
	}`

**Review by shanshuo0918 — APPROVED** [review]
Please squash and merge

**Pull request merged**